capstone-backend/pom.xml

```diff
@@ -37,6 +37,16 @@
       <version>4.12</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-all</artifactId>
+      <version>1.9.5</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
```

Comment:
Is there a specific reason you added Guava to the project? Does it help do Servlets' test?

Reply:
I added Guava so that I could do the @VisibleForTesting on my servlet methods that were originally private. I can ask @VivekShankar more about this to see if it's still necessary.

Reply:
Got it, I forgot to read the main comment. 😄

Reply:
I tried removing VisibleForTesting and Guava and the test still passes with the method `package-private` so I'll leave it out!

Reply:
Nice!